apps/meteor/client/views/admin/subscription/SubscriptionPage.tsx

```diff
@@ -71,79 +57,80 @@ const SubscriptionPage = () => {
 
 	return (
 		<Page bg='tint'>
-			<Page.Header title={t('Subscription')}>{!isRegisteredLoading && getHeaderButtons()}</Page.Header>
+			<Page.Header title={t('Subscription')}>
+				{!isRegisteredLoading && (
+					<ButtonGroup>
+						{isRegistered && subscriptionSuccess && (
+							<Button
+								icon={syncLicenseUpdate.isLoading ? undefined : 'reload'}
+								disabled={syncLicenseUpdate.isLoading}
+								onClick={() => handleSyncLicenseUpdateClick()}
+							>
+								{syncLicenseUpdate.isLoading ? <Throbber size='x12' inheritColor /> : t('Sync_license_update')}
+							</Button>
+						)}
+						<UpgradeButton primary mis={8} i18nKey={isEnterprise ? 'Manage_subscription' : 'Upgrade'} />
+					</ButtonGroup>
+				)}
+			</Page.Header>
 
 			<Page.ScrollableContentWithShadow p={16}>
 				{subscriptionSuccess && (
 					<Callout type='info' title={t('Sync_license_update_Callout_Title')} m={8}>
 						{t('Sync_license_update_Callout')}
 					</Callout>
 				)}
-				{!isLicenseLoading ? (
+				{isLicenseLoading && <SubscriptionPageSkeleton />}
+				{!isLicenseLoading && (
```

Comment:
I know that some people prefer this way, but following our recommended best practices, this is not recommended...
https://alexkondov.com/tao-of-react/#conditional-rendering

Reply:
they suggests ternary to avoid zeros but here we are handling booleans we should be fine :)